scanners/ios/utility.go

```diff
@@ -503,7 +528,14 @@ func GenerateOptions(projectType XcodeProjectType, result DetectResult) (models.
 
 			for _, exportMethod := range exportMethods {
 				// Whether app-clip export Step is added later depends on the used export method
-				configDescriptor := NewConfigDescriptor(project.IsPodWorkspace, project.CarthageCommand, scheme.HasXCTests, scheme.HasAppClip, result.HasSPMDependencies, exportMethod, scheme.Missing)
+				configDescriptor := NewConfigDescriptor(
+					project.IsPodWorkspace,
+					project.CarthageCommand,
+					scheme.HasXCTests,
+					scheme.HasAppClip,
+					result.HasSPMDependencies,
+					project.IsSPMProject,
```

Comment:
here we could implicitly use `false` instead of project.IsSPMProject, to signal that in this case, it isn't an SPM project for sure